Tools/machines/leonardo-cineca/leonardo_gpu_warpx.profile.example

```diff
@@ -0,0 +1,43 @@
+# required dependencies
+module load profile/base
+module load cmake/3.24.3
+module load gmp/6.2.1
+module load mpfr/4.1.0
+module load mpc/1.2.1
+module load gcc/11.3.0
+module load cuda/11.8
+module load zlib/1.2.13--gcc--11.3.0
+module load openmpi/4.1.4--gcc--11.3.0-cuda-11.8
+
+# optional: for QED support with detailed tables
+module load boost/1.80.0--openmpi--4.1.4--gcc--11.3.0
+
+# optional: for openPMD and PSATD+RZ support
+module load openblas/0.3.21--gcc--11.3.0
+export CMAKE_PREFIX_PATH=/leonardo/prod/spack/03/install/0.19/linux-rhel8-icelake/gcc-11.3.0/c-blosc-1.21.1-aifmix6v5lwxgt7rigwoebalrgbcnv26:$CMAKE_PREFIX_PATH
```

Comment:
Good find! Is there a module for `c-blosc` that we can use?

Reply:
Thank you @ax3l for the revisions! No, no module for c-blosc.